docs/source/release_notes.rst

```diff
@@ -8,6 +8,7 @@
         * Fixed ``get_pipelines`` to reset pipeline threshold for binary cases :pr:`3360`
     * Changes
     * Documentation Changes
+        * Fixed documentation links :pr:`3358`
```

Comment:
Mega nit: could this be slightly more explicit? (i.e. "fixed broken documentation links" or "fixed documentation links to point to correct pages", or something like that)

Reply:
Good call, can do!